hydrography/add-enc-exchange-set/src/main/java/com/esri/samples/add_enc_exchange_set/AddEncExchangeSetSample.java

```diff
@@ -63,6 +64,10 @@ public void start(Stage stage) {
       stage.setScene(scene);
       stage.show();
 
+      // authentication with an API key or named user is required to access basemaps and other location services
+      String yourAPIKey = System.getProperty("apiKey");
+      ArcGISRuntimeEnvironment.setApiKey(yourAPIKey);
+
       // create a map view and add it to the stack pane
       mapView = new MapView();
       stackPane.getChildren().add(mapView);
```

Comment:
Should we take the time here and make this pattern consistent with the rest of the samples? I.e. put the part where we add the MapView to the StackPane at the very end of the `start` method?